code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -645,3 +652,5 @@
 	if(.)
 		return
 	return (sunder * -0.01) + 1
+
+up
```

Comment:
????

Reply:
weird, must be a typo/stray copypasta; I'll remove it